### DISCUSSION TITLE
Archery 支持接入第三方用户账号登陆吗

### DISCUSSION THREAD

**Discussion by Bennu-Li** [hhyo/Archery#2119] [Q&A]
想要外接第三方账号管理 Archery 用户账号和权限(最好是 auth0，或者 github 和 email 账号等)，可以怎么做呢

**Comment by LeoQuote** [comment]
master 分支上的代码支持 oauth2 和 oidc

**Reply by Bennu-Li** [reply]
感谢！
有相关的使用文档介绍吗
以及目前发布的最新镜像支持该功能吗

**Reply by LeoQuote** [reply]
现在没有文档, 你可以参考这个 pr https://github.com/hhyo/Archery/pull/2028
